src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/PrisonerSessionValidator.kt

```diff
@@ -1,87 +1,27 @@
 package uk.gov.justice.digital.hmpps.visitscheduler.utils
 
 import org.springframework.stereotype.Component
-import uk.gov.justice.digital.hmpps.visitscheduler.dto.enums.IncentiveLevel
 import uk.gov.justice.digital.hmpps.visitscheduler.dto.prison.api.PrisonerHousingLevels
 import uk.gov.justice.digital.hmpps.visitscheduler.model.entity.session.SessionTemplate
-import java.util.function.Predicate
+import uk.gov.justice.digital.hmpps.visitscheduler.utils.validators.SessionCategoryValidator
+import uk.gov.justice.digital.hmpps.visitscheduler.utils.validators.SessionIncentiveValidator
+import uk.gov.justice.digital.hmpps.visitscheduler.utils.validators.SessionLocationValidator
 
 const val LOCATION_NOT_PERMITTED = -10
 
 @Component
 class PrisonerSessionValidator(
```

Comment:
this is the original class and will eventually be removed once all methods are moved to the Service layer.